index.html

```diff
@@ -1,534 +1,1010 @@
           <p>
-            Asistență în cazuri legate de accidente de vehicule și alte
-            aspecte legate de transport.
+            Asistență în cazuri legate de accidente de vehicule și alte aspecte
+            legate de transport.
           </p>
         </div>
-      </div>
-      <div class="features__feature features__feature-m">
-        <div class="features__icon features__icon-m">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-credit-card"></use>
-          </svg>
-        </div>
-        <div class="features__deatail-cont">
-          <h5 class="features__header">
-            Asistare în materie contractuală
-          </h5>
+        <div class="features__feature">
+          <div class="features__icon">
+            <svg>
+              <use xlink:href="img/icons.svg#icon-credit-card"></use>
+            </svg>
+          </div>
+          <h5 class="features__header">Asistare în materie contractuală</h5>
           <p>
             Ajutor în înțelegerea și negocierea contractelor pentru a proteja
             interesele clienților.
           </p>
         </div>
-      </div>
-      <div class="features__feature features__feature-m">
-        <div class="features__icon features__icon-m">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-credit-card"></use>
-          </svg>
-        </div>
-        <div class="features__deatail-cont">
-          <h5 class="features__header">
-            Reprezentare în materie contractuală
-          </h5>
+        <div class="features__feature">
+          <div class="features__icon">
+            <svg>
+              <use xlink:href="img/icons.svg#icon-credit-card"></use>
+            </svg>
+          </div>
+          <h5 class="features__header">Reprezentare în materie contractuală</h5>
           <p>
             Ajutor în înțelegerea și negocierea contractelor pentru a proteja
             interesele clienților.
           </p>
         </div>
       </div>
-    </div>
-  </section>
+      <div class="features features-m">
+        <img
+          src="img/law-1--lazy.jpg"
+          data-src="img/law-1.jpg"
+          alt="Computer"
+          class="features__img features__img-m lazy-img"
+        />
+        <div class="features__feature features__feature-m">
+          <div class="features__icon features__icon-m">
+            <svg>
+              <use xlink:href="img/icons.svg#icon-monitor"></use>
+            </svg>
+          </div>
+          <div class="features__deatail-cont">
+            <h5 class="features__header">Cabinetul oferă servicii juridice</h5>
+            <p>
+              Avocatul Sebastian Nagy Cabinetul oferă servicii juridice în
+              diverse domenii de drept, adaptate nevoilor specifice
+              dumneavoastră.
+            </p>
+          </div>
+        </div>
+        <img
+          src="img/law-3--lazy.jpg"
+          data-src="img/law-2.jpg"
+          alt="Plant"
```

Comment:
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii legale".